src/SmartTransactionsController.ts

```diff
@@ -420,29 +467,29 @@ export default class SmartTransactionsController extends BaseController<
   // ! Ask backend API to accept list of uuids as params
   async fetchSmartTransactionsStatus(
     uuids: string[],
+    chainId: Hex,
```

Comment:
This is a breaking change — necessary?

Reply:
same as [here](https://github.com/MetaMask/smart-transactions-controller/pull/210#discussion_r1362749950)